src/app/notifications/components/ToastNotifications/Toast.spec.tsx

```diff
@@ -148,18 +142,21 @@ describe('SearchFailure', () => {
 
     renderer.act(() => {
       component.update(<MessageProvider>
-        <SearchFailure dismiss={dismiss} mobileToolbarOpen={false} selectedHighlight={{} as any}  />
+        <Toast dismiss={dismiss} notification={{...toast, timestamp: Date.now() + 10}} positionProps={position} />
       </MessageProvider>);
     });
 
     expect(bannerBody.props.isFadingOut).toBe(false);
     component.unmount();
   });
 
-  describe('syncState', () => {
-    it('doesn\'t allow to fade out if error is not ready to be dimissed', () => {
-      const state = { isFadingOut: false, shouldAutoDismiss: false};
-      expect(syncState(state)).toBe(state);
+  describe('syncAnimationState', () => {
+    it('doesn\'t allow to fade out if toast is not ready to be dimissed', () => {
```

Comment:
I think it would be okay to add to this test also the test case when `prevState.shouldAutoDismiss` is set to `true`